### PR TITLE
Setup notes k8s124up

### PR DESCRIPTION
**Pull request by BnJam**
Addresses Issue #394

### CODE REVIEW

**Review by flaviuvadan — COMMENTED** [review]
Thank you for the contribution @BnJam! I am sure this will help clarify things for many users. Any chance the patch can be removed since this primarily changes the `README`?

**Comment by BnJam** [comment]
Absolutely, I'll take those changes off

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ The general format is:
 
 ```
 
+# 4.3.0 - DATE (30/12/2022)
+
+### Added
+
+- Note added to README for use of authentication method and Bearer token
+
```
Comment (flaviuvadan) [inline CHANGELOG.md:37]: CHANGELOG entries are not necessary for documentation updates :) LGTM otherwise!

```suggestion
```

**Review by flaviuvadan — APPROVED** [review]
Will merge after the CHANGELOG adjustments! 🚀

**Pull request merged**